main-app/MoveMaster-app/Assets/Scripts/Scoreboard/ScoreboardManager.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class ScoreboardManager : MonoBehaviour
+{
+    public static ScoreboardManager Instance;
+    [SerializeField]
+    private TMP_Text scorePoints;
```

Comment:
Dla pól prywatnych camelCase (_scorePoints) -> https://codenga.pl/strony/poradniki/konwencje_nazewnicze_w_jezyku_csharp?srsltid=AfmBOopii31SRMjnHHcPilzoQ0hr6TkAnyrZqN0qXTU8Cj0UdY4HKOJ2

Reply:
Można, ale to już zależy od szkoły

Reply:
Jak chcesz to możesz zmienić na ten koncept